device-types/Arista/DCS-720DP-24ZP.yaml

```diff
@@ -0,0 +1,109 @@
+---
+manufacturer: Arista
+model: DCS-720DP-24ZP
+slug: arista-dcs-720dp-24zp
+part_number: DCS-720DP-24ZP
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+weight: 7
+weight_unit: kg
+comments: >
+  The Arista DCS-720DP-24ZP is a compact 1RU campus aggregation switch featuring 24 x 2.5G RJ45 PoE ports
+  and 4 x 10G SFP+ uplinks. It supports up to 60W PoE per port (802.3bt), making it ideal for high-performance
+  access layer deployments in modern enterprise or campus networks.
+  [Arista CCS-720D Series Datasheet](https://www.arista.com/assets/data/pdf/Datasheets/CCS-720D-Datasheet.pdf)
+
+console-ports:
+  - name: Console
+    type: rj-45
+
+interfaces:
+  - name: Ethernet1
+    type: 2.5gbase-t
+    poe_mode: pse
```

Comment:
For POE interfaces please add the poe_type property e.g. `type3-ieee802.3bt` for 60w poe